src/Knp/DictionaryBundle/Dictionary/Invokable.php

```diff
@@ -30,24 +28,12 @@ final class Invokable implements Dictionary
      */
     private $callable;
 
-    /**
-     * @var array<mixed>
-     */
-    private array $callableArgs = [];
-
     /**
      * @param mixed[] $callableArgs
      */
-    public function __construct(string $name, callable $callable, array $callableArgs = [])
-    {
-        $this->name         = $name;
-        $this->callable     = $callable;
-        $this->callableArgs = $callableArgs;
-    }
-
-    public function getName(): string
+    public function __construct(private string $name, callable $callable, private array $callableArgs = [])
```

Comment:
My PHP skills are a bit rusty ATM but couldn't you use `private callable $callable` here ?

Reply:
Nope, it's not possible yet (I don't know why) to type a property as `callable`. But a method argument, yes.